.gitignore

```diff
@@ -1,4 +1,5 @@
 .*
+!.python-version
```

Comment:
This is unnecessary. `.gitignore` is only considered for files which aren't already in git, and a file matching a pattern here can be added with `git add --force`.

```suggestion
```